src/kontrol/__main__.py

```diff
@@ -72,7 +73,7 @@ def main() -> None:
 
 
 def exec_version(**kwargs: Any) -> None:
-    raise NotImplementedError()
+    print(f'Kontrol Version: {VERSION}')
```

Comment:
```suggestion
    print(f'Kontrol version: {VERSION}')
```
Making the formatting consistent with e.g., how `K` or `kore-rpc` versions are returned:
```
kompile --version
K version:    v6.0.133
```
```
kore-rpc --version
Kore version 0.60.0.0
```